Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -208,6 +208,8 @@ final class PurchasesOrchestrator {
         Task {
             await syncDiagnosticsIfNeeded()
         }
+
+        Logger.debug(Strings.storeKit.sk_storefront_country_code(countryCode: systemInfo.storefront?.countryCode))
```

Comment:
In what cases would this be null? Just asking in case we need to wait to obtain it slightly...

Reply:
Right now it's using SK1 (see [documentation](https://developer.apple.com/documentation/storekit/skpaymentqueue/storefront)), which is a synchronous API. That said, the documentation does recommend calling this API from a background thread, so I will modify it just in case.

I honestly don't know under what circumstances it returns nil. I haven't seen it documented.

I do intend to migrate `systemInfo.storefront` to SK2's Storefront which is indeed asynchronous. But I'll do that in a separate PR because it's likely to imply other changes.

Thanks for the observations!